js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_zh_CN.properties

```diff
@@ -282,6 +282,7 @@ samlKeysExportError=无法导出密钥，因为：{{error}}
 webAuthnPolicyCreateTimeout=超时
 comparison=对比
 deletedSuccessClientScope=客户端作用域已删除
+notAllowedToDeleteAllClientScopes=您不能删除所有客户端范围
```

Comment:
@xianli123 could you verify this translation is accurate?

Reply:
Yes. I made a suggestion for the line #285. @jonkoops